src/app/client/src/app/modules/core/components/content-player-metadata/content-player-metadata.component.html

```diff
@@ -239,5 +239,22 @@
                 </div>
             </div>
         </div>
+        <div class="item padding-top-10 padding-bottom-5">
+            <div class="ui grid">
+                <div class="three wide column">
+                    <div class="content content-detail">
+                        {{resourceService.frmelmnts.lbl.ownership}}
+                    </div>
+                </div>
+                <div class="nine wide column">
+                    <!-- *ngIf="metadata.owner" -->
+                    <div class="content" >
+                        sdfsdfsdfdsf {{metadata.owner}} <a (click)="showContentCredits()">Content Credits</a>
```

Comment:
why  unnecessary data is in before data binding and what happens when credits or owner are not available

Reply:
this change while testing i kept it and missed u remove it.
content credits ui will be shown only when owner is available
when there are no content credits in popup only creator will be shown